data/src/main/scala/ch.epfl.scala.index.data/project/ProjectForm.scala

```diff
@@ -15,9 +20,25 @@ case class ProjectForm(
     cliArtifacts: Set[String],
     // documentation
     customScalaDoc: Option[String],
-    documentationLinks: List[(String, String)]
+    documentationLinks: List[(String, String)],
+    beginnerIssuesLabel: Option[String],
+    chatroom: Option[Url]
 ) {
-  def update(project: Project): Project = {
+  def update(project: Project, paths: DataPaths): Project = {
+    val oldBeginnerIssueLabel = project.github.map(_.beginnerIssuesLabel).getOrElse(None)
+    val beginnerIssues = if (beginnerIssuesLabel != oldBeginnerIssueLabel) {
+      implicit val system = ActorSystem()
```

Comment:
[Done](https://github.com/scalacenter/scaladex/pull/432/commits/0d0e08ce6dbd290c746345aa8c56493a424dcd08#diff-9b0b9f250047a450506bda2259c11537), passed in `GithubDownload` to calls to `update`. Also see my [comment](https://github.com/scalacenter/scaladex/pull/432#issuecomment-311820363) on the PR